apps/aetx/src/aetx.erl

```diff
@@ -28,6 +28,8 @@
 -export([tx/1]).
 -endif.
 
+-define(MAXINT, 16#FFFFffffFFFFffff).
```

Comment:
Interesting formatting hack ❤️

Reply:
Not invented here, saw it elsewhere and thought it improved readability!